contrib/babelfishpg_common/src/collation.c

```diff
@@ -549,6 +549,7 @@ collation_is_accent_insensitive(int collidx)
 		return false;
 
 	if (coll_infos[collidx].collateflags == 0x000f ||	/* CI_AI  */
+		coll_infos[collidx].collateflags == 0x000c ||	/* CS_AS  */
```

Comment:
lets remove this function `collation_is_accent_insensitive` and introduce new one say `collation_is_ci_as`

Reply:
removed function `collation_is_accent_insensitive` and added function `collation_is_case_insensitive_and_accent_sensitive`